### PR TITLE
Bump cassandra driver version from 3.6.0 to 3.11.2

### PR DESCRIPTION
**Pull request by Shoothzj**
### Motivation
that's a regular update, cassanrda driver 3.6.0 is released in year 2018. Keep update.

### CODE REVIEW

**Review by RobertIndie — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -150,7 +150,7 @@ flexible messaging model and an intuitive client API.</description>
     <gson.version>2.8.9</gson.version>
     <sketches.version>0.8.3</sketches.version>
     <hbc-core.version>2.2.0</hbc-core.version>
-    <cassandra-driver-core.version>3.6.0</cassandra-driver-core.version>
+    <cassandra-driver-core.version>3.7.2</cassandra-driver-core.version>
```
Comment (nicoloboschi) [inline pom.xml:153]: it seems that cassandra-driver-core.version and cassandra.version can be merged into a single property, isn't it ?

**Review by nicoloboschi — CHANGES_REQUESTED** [review]
can you drop on of them ? there

File: pulsar-io/cassandra/pom.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@
     <dependency>
       <groupId>com.datastax.cassandra</groupId>
       <artifactId>cassandra-driver-core</artifactId>
-      <version>${cassandra-driver-core.version}</version>
+      <version>${cassandra.version}</version>
```
Comment (nicoloboschi) [inline pulsar-io/cassandra/pom.xml:54]: ia this needed? we already have it on the dependencyManagement section

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

**Review by nicoloboschi — APPROVED** [review]

**Pull request merged**